tox.ini

```diff
@@ -66,4 +66,4 @@ deps =
     -r {toxinidir}/requirements.txt
     -r {toxinidir}/tests/integration/requirements.txt
 commands =
-    functest-run-suite --keep-faulty-model {posargs}
+    functest-run-suite {posargs}
```

Comment:
why do we need this though? I thought we don't want to keep faulty model by default (`make integration` does not have `posargs` so `--keep-faulty-model` will be set)

Reply:
It should be set in our CI. There is and plan to add steps to collect juju-crashdump and then remove model if it's there. We need to remove model only in self-hosted, or add some clean up process in our CI infrastructure in self-hosted.

Reply:
is the step ready now? If it's not, we think we should not keep this for now because it's very easy to break our self hosted runner with this integration test

Reply:
How it could break it? I will add some cleaning process to CI this week, for now I'm doing it only manly.  If it's not there we need to go back to the tox and change it, so that's why I think it's not good to change it to many times, since now it's working fine.

Reply:
Yeah, we should do what you are doing in the other project: add concurrency to the workflow to avoid multiple jobs running at the same time. Now, we don't so `--keep-faulty-model` will used up all resources if one push multiple times

Reply:
Concurrency issue PR created: https://github.com/canonical/openstack-exporter-operator/pull/51